src/components/Calculator.js

```diff
@@ -5,117 +5,136 @@ import Col from 'react-bootstrap/Col';
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               %
             </Button>
           </Col>
           <Col>
-            <Button variant="warning" className="w-100">
-              /
+            <Button variant="warning" className="w-100" onClick={this.handleClick}>
+              ÷
             </Button>
           </Col>
         </Row>
         <Row>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               7
             </Button>
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               8
             </Button>
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               9
             </Button>
           </Col>
           <Col>
-            <Button variant="warning" className="w-100">
-              *
+            <Button variant="warning" className="w-100" onClick={this.handleClick}>
+              x
             </Button>
           </Col>
         </Row>
         <Row>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               4
             </Button>
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               5
             </Button>
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               6
             </Button>
           </Col>
           <Col>
-            <Button variant="warning" className="w-100">
+            <Button variant="warning" className="w-100" onClick={this.handleClick}>
               -
             </Button>
           </Col>
         </Row>
         <Row>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               1
             </Button>
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               2
             </Button>
           </Col>
           <Col>
-            <Button variant="secondary" className="w-100">
+            <Button variant="secondary" className="w-100" onClick={this.handleClick}>
               7
             </Button>
```

Comment:
- [ ] Please, consider fixing this button functionality - here should be 3, not 7
![image](https://user-images.githubusercontent.com/49064106/190571751-f0565684-0548-453c-b2b9-eb4ccbac0e16.png)